### ISSUE TITLE
TypeError: Cannot read property 'app' of undefined

### ISSUE THREAD

**Issue by Yingi** [benwinding/react-admin-firebase#35]
I cloned your project, followed it has it is on Your README. But I am getting this error this error

Cannot read property 'app' of undefined

src/providers/database/firebase/FirebaseWrapper.ts:34:14
On line 34 >  if (options.app) {

src/providers/database/firebase/FirebaseWrapper.ts:16:33

On line 16 >     this.app = ObtainFirebaseApp(firebaseConfig, options) as any;

The List goes on. What could be the issue and possible solution

I used the my firebase config like you specified, But still having the same error

**Comment by benwinding** [comment]
Hi @Yingi,
Thanks for checking out the project! This shouldn't have happened and will be fixed shortly.

**Fix using an empty object for options**
``` javascript
const options = {}; // Make sure this is at least an empty object
const authProvider = FirebaseAuthProvider(firebaseConfig, options);
const dataProvider = FirebaseDataProvider(firebaseConfig, options);
const firebaseRealtime = FirebaseRealTimeSaga(dataProvider, options);
```

Cheers,
Ben

**Comment by benwinding** [comment]
Should be fixed with version `0.8.11` :+1:
